lib/commands/ci.js

```diff
@@ -79,11 +87,14 @@ class CI extends ArboristWorkspaceCmd {
       // Only remove node_modules after we've successfully loaded the virtual
       // tree and validated the lockfile
       await time.start('npm-ci:rm', async () => {
-        const path = `${where}/node_modules`
-        // get the list of entries so we can skip the glob for performance
-        const entries = await fs.readdir(path, null).catch(() => [])
-        return Promise.all(entries.map(f => fs.rm(`${path}/${f}`,
-          { force: true, recursive: true })))
+        return await Promise.all([where, ...this.workspacePaths].map(async modulePath => {
```

Comment:
```suggestion
        return await Promise.all(workspaces.values().map(async modulePath => {
```

We don't also need `where` in here since we asked `getWorkspaces` to include the workspace root.